cms/jinja2/templates/pages/statistical_article_page--previous-releases.html

```diff
@@ -1,28 +1,80 @@
 {% extends "templates/base_page.html" %}
 
+{% from "components/document-list/_macro.njk" import onsDocumentList %}
+{% from "components/hero/_macro.njk" import onsHero %}
+{% from "components/pagination/_macro.njk" import onsPagination %}
+
+{% block header_area %}
+    {%- set hero_topic = _("Previous Releases") -%}
+    {%- set hero_title = _("Previous Releases") -%} {# TODO: Needs to be updated to use related Statistical Article title #}
+    {%- set breadcrumbs_aria_label = _("Breadcrumbs") -%}
+
+    {# fmt:off #}
+    {{
+        onsHero({
+            "topic": hero_topic,
+            "title": hero_title,
+            "variants": 'grey',
+            "text": page.summary,
+            "wide": true,
+            "detailsColumns": 12,
+            "breadcrumbs": {
+                "ariaLabel": breadcrumbs_aria_label,
+                "itemsList": breadcrumbs(page, include_self=True),
+            }
+        })
+    }}
+    {# fmt:on #}
+{% endblock %}
+
 {% block main %}
-    <h1>{{ _("Previous Releases") }}</h1>
-    <ul>
-        {% for page in pages %}
-            <li><a href="{{ pageurl(page) }}">{{ page.title }}</a> - {{ page.release_date }}</li>
-        {% else %}
-            <p>{{ _("There are currently no releases") }}</p>
-        {% endfor %}
-    </ul>
-
-    {% block pagination %}
-        {% from "components/pagination/_macro.njk" import onsPagination %}
-        {% if pages.paginator.num_pages > 1 %}
-
-{# fmt:off #}
-            {{
-            onsPagination({
-            "currentPageNumber": pages.number ,
-            "pages": ons_pagination_url_list
-            })
-            }}
-        {# fmt:on #}
-        {% endif %}
-    {% endblock %}
+    <div class="ons-grid">
+        <div class="ons-grid__col ons-col-8@m">
+            {%- set releases = namespace(previous_release_pages=[]) -%}
```

Comment:
question: is this because of Jinja's loop isolation?
if so, this could use a developer note

Reply:
Yes it is - good shout, have added a note here 👍